package.json

```diff
@@ -135,17 +135,17 @@
     "@types/redis": "^2.8.32",
     "@types/superagent": "^4.1.13",
     "@types/supertest": "^2.0.11",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
+    "@typescript-eslint/eslint-plugin": "^5.10.2",
```

Comment:
Upped the version of the linter in the hope that it would fix emum issues - it didn't but I figure being at the latest is probably worth leaving in